app/services/hyrax/migrator/crosswalk_metadata_service.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal:true
+
+require 'rdf'
+
+module Hyrax
+  module Migrator
+    # Called by the CrosswalkMetadataActor to map OD1 metadata to OD2
+    class CrosswalkMetadataService
+      include RDF
+
+      CONFIG_FILE = 'config/migrator/crosswalk.yml'
```

Comment:
I feel like we should consider allowing the crosswalk yaml to be passed into the service initializer so that it's possible to specify a crosswalk upstream from this actor/service.